cast/src/lib.rs

```diff
@@ -676,8 +676,13 @@ where
             .ok_or_else(|| eyre::eyre!("tx not found: {:?}", tx_hash))?;
 
         Ok(if let Some(ref field) = field {
-            get_pretty_tx_attr(&tx, field)
-                .ok_or_else(|| eyre::eyre!("invalid tx field: {}", field))?
+            match field.as_ref() {
+                "raw" => {
```

Comment:
I'd also like to support `--raw` if that's a valid value via cli

Reply:
Just to confirm, we want to have `--raw` for incase `raw` field actually exists in the response? 

Logic would then be:
1. Check for `--raw`, if true then output `tx.rlp()` and return
2. If "raw" field is specified, check for presence of `raw` in tx and if present return as is, else fallback to `tx.rlp()`

Reply:
just treat `--raw` and `raw` as field values as equivalent like

`"raw" | "--raw"`